WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -411,6 +415,17 @@ extension InteractiveNotificationsManager: UNUserNotificationCenterDelegate {
         completionHandler([])
     }
 
+    private func handleZendeskNotification(userInfo: NSDictionary) {
+        if FeatureFlag.zendeskMobile.enabled {
+            if let type = userInfo.string(forKey: ZendeskUtils.PushNotificationIdentifiers.key),
```

Comment:
Bit of a nitpick, but in the interest of less nesting maybe return early instead?  Up to you.